attributes/default.rb

```diff
@@ -14,8 +14,7 @@
 default['logstash']['version']                               = "7.2.0"
 default['logstash']['url']                                   = "#{node['download_url']}/logstash-oss-#{node['logstash']['version']}.tar.gz"
 default['logstash']['beats']['spark_port']                   = "5044"
-default['logstash']['beats']['serving_tf_port']              = "5045"
-default['logstash']['beats']['serving_sklearn_port']         = "5047"
+default['logstash']['beats']['serving_port']                 = "5046"
```

Comment:
Yes, instead of having one filebeat container per pod, a daemonset is created which sends the container logs to the logstash pipeline. These logs include the pod labels running the containers. I use these labels (e.g `serving.hops.works/model_server`) to parse the logs accordingly in the logstash pipeline.
Not all containers logs are sent to logstash, I added a filter in [filebeat config](https://github.com/logicalclocks/kube-hops-chef/pull/34/files/de9677b9a780a2aadf44dd65f5096692073e7a43#diff-e00340573d1e74540cd7b2f222fbd4c2c771aba056ec79bbd378835bea4364e2) that checks for the existence of `serving.hops.works/id`, only present in containers deployed by Hopsworks to serve models.